src/processing/resources/index.ts

```diff
@@ -43,13 +43,13 @@ function getResourceUrlReplacer (ctx: RequestPipelineContext): Function {
 }
 
 export async function process (ctx: RequestPipelineContext): Promise<Buffer> {
-    const body        = ctx.destResBody;
-    const contentInfo = ctx.contentInfo;
-    const encoding    = contentInfo.encoding;
-    const charset     = contentInfo.charset;
+    const { destResBody: body, contentInfo } = ctx;
+    const { encoding, charset }              = contentInfo;
 
     const decoded = await decodeContent(body, encoding, charset);
 
+    await ctx.prepareInjectableUserScripts();
```

Comment:
This function has an impact only for page resources and this place wrong for this call.